main/integration-tests/__test__/api-tests/studies/__fixtures__/update-study-fixture.js

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const BaseFixture = require('../../../../helpers/base-fixture');
+const { getInternalUserClient } = require('../../../../utils/auth-tokens');
+const { createStudyJson, createStudy } = require('../../../../utils/studies');
+const { createUserJson, createUser } = require('../../../../utils/users');
+
+// Test Fixture
+/**
+ * This class is designed to pre-generate & help generate (using methods) resources
+ * that would be required by a SPECIFIC integration test suite
+ *
+ * The verification of these components are performed once every respective test suite cycle
+ * For example, whenever update-study-api test suite is triggered
+ */
+class UpdateStudyFixture extends BaseFixture {
+  constructor() {
+    super();
+    // This static property keeps track of whether or not its parent needs a check
+    // since parent may be different than BaseFixture
+    UpdateStudyFixture.baseReady = BaseFixture.ready;
+  }
+
+  async setupParent() {
+    await this.setupBasePreRequisites();
+    UpdateStudyFixture.baseReady = BaseFixture.ready;
+  }
+
+  // IMPORTANT: These will be created afresh every new integration test cycle
+  // Create and add pre-requisite resources here as needed
+  async setupPreRequisites() {
+    // We currently only need helper methods to assist tests in the update-study test suite
+    // Since we don't have any pre-requisite resources needed, set this to true
+    UpdateStudyFixture.ready = true;
+  }
+
+  async createNonAdminUser(axiosClient, projectId) {
+    const testName = 'UpdateStudy';
+    const nonAdminUserJson = createUserJson({ projId: projectId, testName });
```

Comment:
nitpick: When building the JSON and not making a request, maybe we can use the term "build". In this case it would be `buildUserJson`

Reply:
Fixed. Coming in the next commit